### ISSUE TITLE
Unable to Change Theme in Magento 2.3.0 (Non-duplicate, Requested to Open as a New Issue)

### ISSUE THREAD

**Issue by Microtribute** [magento/magento2#21669]
### Preconditions (*)
- Magento Community Edition 2.3.0
- WAMP 3.1.7 with PHP 7.1.26
- Windows Server 2019 Datacenter.
- Google Chrome Version 72.0.3626.121 (Official Build) (64-bit)

### Steps to reproduce (*)
1. Log into the Admin Console.
2. Go to Content -> Configuration
3. Click an Edit link from any of the rows of the displayed table of stores and store views.
4. Change the Applied Theme to something else.
5. Click the 'Save Configuration' button

### Expected result (*)
The minimum expected result would be the change is saved and the user gets taken back to the previous page - Design Configuration.

### Actual result (*)
1. Neither the 'Save and Config' button nor the orange 'Save Config' button does the expected result.
2. Nothing happens just staying on the same page.
3. Getting back to the design configuration page shows the old value.
4. There are no JavaScript errors logged in the console (which i was expecting at least in the event the buttons do not work.)

**Comment by magento-engcom-team** [comment]
Hi @nightlybuilt. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@nightlybuilt do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by vasilii-b** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @vasilii-b. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @vasilii-b, here is your [Magento instance](https://i-21669-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21669-2-3-develop.instances.magento-community.engineering/admin](https://i-21669-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by DanielRuf** [comment]
@Nazar65 why the downvote?

**Comment by Nazar65** [comment]
@DanielRuf WAMP 3.1.7 with PHP 7.1.26

**Comment by DanielRuf** [comment]
Not sure but this should be no problem.

The PHP version is supported and you can use the stack you want - if it works is a different thing.

**Comment by Nazar65** [comment]
@DanielRuf WAMP 3.1.7 i'm mean that windows not supports

**Comment by DanielRuf** [comment]
I am aware of this. But please let us check if we can reproduce this or not. Someone is already working on this.

**Comment by DanielRuf** [comment]
We just dropped the official Windows support due to specific issues. This does not mean that Magento does not run on Windows anymore.

**Comment by Microtribute** [comment]
@DanielRuf @Nazar65 I don't think the issue is happening because I am using WAMP or Windows. Other parts of Magento work well for me.

**Comment by magento-engcom-team** [comment]
Hi @Leandry. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by Leandry** [comment]
Hi, @nightlybuilt
I was not able to reproduce the issue. I successfully changed the theme and saved the store view/website.
![2019-03-10_20-44-51](https://user-images.githubusercontent.com/9416636/54089849-1f30bb00-4376-11e9-8ef2-046051687799.gif)


Could you try to reproduce the issue on Vanilla Magento 2.3 CE instance?
`@magento-engcom-team give me 2.3-develop instance`

Thank you!

**Comment by magento-engcom-team** [comment]
Hi @Leandry. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Leandry, here is your [Magento instance](https://i-21669-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21669-2-3-develop.instances.magento-community.engineering/admin](https://i-21669-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by Microtribute** [comment]
Guys, I use 2.3.0 not 2.3-develop.

**Comment by DanielRuf** [comment]
> Guys, I use 2.3.0 not 2.3-develop.

So it is solved in the latest dev build then.
So this will be fixed in 2.3.1 or later.

**Comment by DanielRuf** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @DanielRuf. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @DanielRuf, here is your [Magento instance](https://i-21669-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-21669-2-3-0.instances.magento-community.engineering/admin](https://i-21669-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by Microtribute** [comment]
> > Guys, I use 2.3.0 not 2.3-develop.
> 
> So it is solved in the latest dev build then.
> So this will be fixed in 2.3.1 or later.

@DanielRuf How do I install Magento 2.3.1? Can't find an official documentation. Just do a git clone?

**Comment by DanielRuf** [comment]
I can not reproduce it on 2.3.0.

**Comment by DanielRuf** [comment]
> @DanielRuf How do I install Magento 2.3.1? Can't find an official documentation. Just do a git clone?

2.3.1 is not yet released.
https://github.com/magento/magento2/releases

**Comment by Leandry** [comment]
Not reproduce in 2.3.0

**Comment by Microtribute** [comment]
@Leandry @DanielRuf @vasilii-b et al

This happens after I migrated from the old Magento 1.7.0.2 site. The same codebase works well with the sample data but not with the migrated data. What could be possibly the issue?

**Comment by DanielRuf** [comment]
> This happens after I migrated from the old Magento 1.7.0.2 site

You migrated directly from 1.7 to 2.3? Not sure if this is even supported.

**Comment by Microtribute** [comment]
of course i used the official magento migration tool.

2019/3/12 14:34，Daniel Ruf <notifications@github.com>寫道：

This happens after I migrated from the old Magento 1.7.0.2 site

You migrated directly from 1.7 to 2.3? Not sure if this is even supported.

**Comment by DanielRuf** [comment]
Not sure. But then this is a different scenario which was not mentioned in your initial issue description so we could not reproduce this.

**Comment by Microtribute** [comment]
Turns out there was an invalid value with the watermarking configuration (opacity). Funny thing is the opacity value is "Enter your city here" which is not a number. That was causing a data validation error and the configuration could not be saved. As the items of the accordion were all collapsed, the error was not visible. Maybe in the future, if an error found in the form, we could open the accordion and scroll down to the part where the error is occurring - in order to give the user a visually better feedback.

**Comment by DanielRuf** [comment]
Hi @nightlybuilt,

Thanks for the feedback. Is there still the wrong placeholder in develop-2.3 and could we improve / add checks on submission regarding the watermarking configuration?

**Comment by Microtribute** [comment]
Hi @DanielRuf et al, 
That was a migrated value, not a placeholder.

**Comment by Microtribute** [comment]
Sincere thank you to all of you for supporting me and taking the time to solve the issue! What an amazing community!
